### ISSUE TITLE
binder build fails with Error during build: .0.0 is not valid SemVer string

### ISSUE THREAD

**Issue by cchwala** [OpenSenseAction/OPENSENSE_sandbox#1]
This is the error message when trying to run binder

```
Waiting for build to start...
Picked Git content provider.
Cloning into '/tmp/repo2dockerzodpbb8d'...
HEAD is now at acf2947 Very first try with some Python and R packages in a conda-env and a binder link
Submodule 'RAINLINK' (https://github.com/overeem11/RAINLINK.git) registered for path 'RAINLINK'
Submodule 'pycomlink' (https://github.com/pycomlink/pycomlink) registered for path 'pycomlink'
Cloning into '/tmp/repo2dockerzodpbb8d/RAINLINK'...
Cloning into '/tmp/repo2dockerzodpbb8d/pycomlink'...
Submodule path 'RAINLINK': checked out '1a7677affe681397444f5c65697c5d7e7c2b61c4'
Submodule path 'pycomlink': checked out '12fc302539851b19f7656cf7e2438c0ddbaa48bf'
Error during build: .0.0 is not valid SemVer string
```

**Comment by cchwala** [comment]
a solution might be here https://github.com/jupyterhub/repo2docker/issues/1140#issuecomment-1050755843

> it looks like a (the?) problem is that `r-base` needs to be pinned to a specific version; I had it in `binder/environment.yml` as just `r-base`, and when I set it to `r-base=4.0.5` it started building.

**Comment by cchwala** [comment]
fixed in a5451f82544de0308705918a04afc9314fa43876
